falcon/request_helpers.py

```diff
@@ -38,6 +44,63 @@ def fget(self):
     return property(fget)
 
 
+def make_etag(value, is_weak=False):
```

Comment:
This function should be private IMHO, or we if we need to make it public, we need to document its parameters.

Furthermore, it is used just twice, so it could be inlined instead.

Reply:
This function also imported in other source file and probably it would be a good builder for our public class `ETag`.